src/sass/grid/_print.scss

```diff
@@ -225,3 +225,13 @@ screen and (min-width: 700px) {
 }
 
 // End of adjustment to get page breaks to work in multi-line flexbox
+
+// Another workaround to get page breaks to works
+// https://github.com/OpenClinica/enketo-express-oc/issues/463
+.or {
+    &:not(.print-relevant-only) section.question.or-branch.disabled, 
+    &:not(.print-relevant-only) section.or-branch.disabled {
+        display: table;
```

Comment:
Why `display: table` if we don't use that for any other questions, e.g. non-disabled questions. Why not use the same solution for everything? I cannot think of any reason why we would use a different solution for disabled vs. non-disabled.

Reply:
We still have some custom solution for branches. It makes no sense to do this just for branches. I guarantee that either:

a. the bug still exists for other sections/question that are not a branch (have no relevant), or
b. that `display: table` is not necessary

But I'm getting tired of this now, and OC is waiting, so I'll just approve and merge this.

Reply:
I'm sorry for the inconvenience